scripts/benchmarks/benches/compute.py

```diff
@@ -355,3 +355,30 @@ def bin_args(self) -> list[str]:
             f"--SrcUSM={self.srcUSM}",
             f"--DstUSM={self.dstUSM}",
         ]
+
+class GraphApiSinKernelSYCL(ComputeBenchmark):
+    def __init__(self, bench):
+        super().__init__(bench, "graph_api_benchmark_sycl", "SinKernel")
+
+    def name(self):
+        return f"graph_api_benchmark_sycl SinKernel"
+
+    def bin_args(self) -> list[str]:
+        return [
+            "--iterations=1000",
+            "--numKernels=100",
```

Comment:
should we add more scenarios with different number of kernels? e.g., with 1 kernel, to see the cost of the whole machinery end-to-end.